src/java.base/share/classes/java/nio/charset/spi/CharsetProvider.java

```diff
@@ -36,7 +36,7 @@
  * zero-argument constructor and some number of associated charset
  * implementation classes.  Charset providers may be installed in an instance
  * of the Java platform as extensions.  Providers may also be made available by
- * adding them to the applet or application class path or by some other
+ * adding them to the application class path or by some other
```

Comment:
They can also be deployed on the application module path, maybe need a separate JBS issue to clarify that part of the spec, and add a test.

Reply:
Filed https://bugs.openjdk.org/browse/JDK-8340404 to handle this.